include/dds/core/xtypes/detail/MemberType.hpp

```diff
@@ -82,15 +80,7 @@ class MemberType
 
     void add_annotation(const xtypes::Annotation& annotations)
```

Comment:
Lexically, you are adding a single annotation

Reply:
```suggestion
    void add_annotation(const xtypes::Annotation& annotation)
```

Reply:
Already fixed

Reply:
Also fixed the parameter style